web/src/utils/router.ts

```diff
@@ -73,3 +75,11 @@ export const removeQueryArguments = (keys: AnyQueryArgs[]) => {
     history.replace(`${url.pathname}?${params.toString()}${url.hash}`);
   }
 };
+
+export const navigate = (pathName: string, keepParams = true) => {
```

Comment:
it seems that this `navigate` triggers a full reload of the console, if possible we should probably avoid that (but i don't know if there are other options)

Reply:
(I think this is what react-router is for)

Reply:
Thanks, I used 'react-router-dom' `useHistory` hook :+1: